tap_pendo/streams.py

```diff
@@ -369,8 +370,6 @@ def is_selected(self):
     def transform(self, record):
         return humps.decamelize(record)
 
-    LOG_PROGRESS_PERCENTAGE_INTERVAL = 5
-
     def sync_substream(self, state, parent, sub_stream, parent_response):
```

Comment:
There are now multiple loops inside this method. Would it be more readable if it was broken up? Maybe make each for loop a method that is called inside sync_substream? What do you think?

Reply:
I would caution too much refactoring. This is to solve a memory issue and this change is a propagation from that change. Pulling the refactor to readability standards thread might unravel the whole sweater and muddy up the effectiveness of the original issue.

Reply:
Fair enough